lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -175,6 +175,8 @@ def define
               outcome :outcome_opposite_sex_marriage_in_poland_when_residing_in_uk_or_poland
             elsif calculator.ceremony_country == 'slovenia'
               outcome :outcome_opposite_sex_marriage_in_slovenia_when_residing_in_uk_or_slovenia
+            elsif calculator.ceremony_country == 'denmark' && (calculator.resident_of_uk? || calculator.resident_of_ceremony_country?)
+              outcome :outcome_ceremonies_in_denmark_when_residing_in_uk_or_denmark
```

Comment:
This is duplicated for opposite and same sex. I think I'd be inclined to move it outside of the checks for opposite and same sex so that it only occurs once.